helm/blog-app/values.yaml

```
@@ -0,0 +1,234 @@
+    initialDelaySeconds: 5
+    periodSeconds: 5
+    timeoutSeconds: 3
+    successThreshold: 1
+    failureThreshold: 3
+    
+  # Security context
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 1001
+    fsGroup: 1001
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)